examples/pytorch/mmcr.py

```diff
@@ -0,0 +1,92 @@
+# Note: The model and training settings do not follow the reference settings
+# from the paper. The settings are chosen such that the example can easily be
+# run on a small dataset with a single GPU.
+
+import copy
+
+import torch
+import torchvision
+from torch import nn
+
+from lightly.loss import MMCRLoss
+from lightly.models.modules import SimCLRProjectionHead
+from lightly.models.utils import deactivate_requires_grad, update_momentum
+from lightly.transforms.mmcr_transform import MMCRTransform
+from lightly.utils.scheduler import cosine_schedule
+
+
+class MMCR(nn.Module):
+    def __init__(self, backbone):
+        super().__init__()
+
+        self.backbone = backbone
+        self.projection_head = SimCLRProjectionHead(512, 512, 128)
```

Comment:
This reminds me that we should also add an `MMCRProjectionHead` with default dimensions (8192, 8192, 512). But this can be done in a different PR.

Reply:
Made an issue for it: https://github.com/lightly-ai/lightly/issues/1481